CONTRIBUTING.md

```diff
@@ -37,7 +37,7 @@ run the following command in the project main directory prior to finishing a PR:
 If encountering CI errors in CircleCI, it may be helpful to execute the test jobs
 locally to debug. This can be facilitated by executing the command
 
-   circleci local execute --job some-test-job
+   circleci local execute some-test-job
```

Comment:
Don't understand this change: Below's the output of `circleci local execute --help`:

```
Run a job in a container on the local machine


Usage:
  circleci local execute [flags]

Flags:
      --branch string         Git branch
      --checkout-key string   Git Checkout key (default "~/.ssh/id_rsa")
  -c, --config string         config file (default ".circleci/config.yml")
  -e, --env -e VAR=VAL        Set environment variables, e.g. -e VAR=VAL
  -h, --help                  help for execute
      --index int             node index of parallelism
      --job string            job to be executed (default "build")
      --node-total int        total number of parallel nodes (default 1)
  -o, --org-slug string       organization slug (for example: github/example-org), used when a config depends on private orbs belonging to that org
      --repo-url string       Git Url
      --revision string       Git Revision
      --skip-checkout         use local path as-is (default true)
  -v, --volume stringArray    Volume bind-mounting

Global Flags:
      --host string         URL to your CircleCI host, also CIRCLECI_CLI_HOST (default "https://circleci.com")
      --skip-update-check   Skip the check for updates check run before every command.
      --token string        your token for using CircleCI, also CIRCLECI_CLI_TOKEN

```